schema/verify/tables/funding_parameter_002_add_auto_generating_report_date_columns.sql

```diff
@@ -0,0 +1,13 @@
+-- Verify cif:tables/funding_parameter_002_add_auto_generating_report_date_columns on pg
+
+begin;
+
+select column_name
+from information_schema.columns
+where table_schema='cif' and table_name='funding_parameter' and column_name='contract_start_date';
+
+select column_name
+from information_schema.columns
+where table_schema='cif' and table_name='funding_parameter' and column_name='project_assets_life_end_date';
```

Comment:
These will not fail but merely return `null` if the column doesn't exist, and won't cause the `verify` script to fail.

Something like that would fail instead:
```sql
do $$
  begin
    assert (
      select column_name
      from information_schema.columns
      where table_schema='cif' and table_name='funding_parameter' and column_name='contract_start_date'
    ), 'column "contract_start_date" is not defined on table "funding_parameter"';
  end;
$$;
```

Reply:
>  I think we need to change the `select column_name` to `select true`

yes, that's right

For the refactor, we can probably do this in a separate PR but that would be good :)